console - js errors

Encountered on chromium / chrome, but the same appears with firefox (https://dev.geo2france.fr/console/account/new):

```
angular.js:15570 Error: [$location:nobase] $location in HTML5 mode requires a <base> tag to be present!
https://errors.angularjs.org/1.7.9/$location/nobase
    at angular.js:138
    at $LocationProvider.$get (angular.js:15221)
    at Object.invoke (angular.js:5143)
    at angular.js:4932
    at getService (angular.js:5086)
    at injectionArgs (angular.js:5111)
    at Object.invoke (angular.js:5135)
    at Object.enforcedReturnValue [as $get] (angular.js:4978)
    at Object.invoke (angular.js:5143)
    at angular.js:4932
```

The map on the new account page does not work afterwards, and the RGPD delete my account feature is broken probably for the same reason.

Also, if I load the "/console/account/userdetails", then I click on the "save" button (without modifying anything), the newly loaded page does not show the map anymore, the RGPD menu is still available, but without the "delete my account option".

![Screenshot from 2020-10-09 12-41-44](https://user-images.githubusercontent.com/594335/95574452-41485e80-0a2d-11eb-9d1d-8f7adfd85cdc.png)



![Screenshot from 2020-10-09 12-41-51](https://user-images.githubusercontent.com/594335/95574317-0c3c0c00-0a2d-11eb-897b-a4df8452ba46.png)

--- a/console/src/main/java/org/georchestra/console/ws/edituserdetails/EditUserDetailsFormController.java
+++ b/console/src/main/java/org/georchestra/console/ws/edituserdetails/EditUserDetailsFormController.java
@@ -181,6 +181,8 @@ public class EditUserDetailsFormController {
             response.sendError(HttpServletResponse.SC_FORBIDDEN);
         }
 
+        model.addAttribute("gdprAllowAccountDeletion", gdprAllowAccountDeletion);
+
         // Validate first name and surname
         validation.validateUserFieldWithSpecificMsg("firstName", formBean.getFirstName(), resultErrors);
         validation.validateUserFieldWithSpecificMsg("surname", formBean.getSurname(), resultErrors);

